aquarius/events/events_monitor.py

```diff
@@ -172,7 +174,12 @@ def process_current_blocks(self):
             self.retry_mechanism.process_queue()
 
         last_block = self.get_last_processed_block()
-        current_block = self._web3.eth.block_number
+        current_block = None
+        try:
+            current_block = self._web3.eth.block_number
```

Comment:
depending on the congestion of the network this can consistenly fail, as block_number usually returns a promise. Coroutines in python could solve this. What do you think?